docs/sphinx/overview.rst

```diff
@@ -0,0 +1,29 @@
+OME Files C++ overview
+======================
+
+OME Files C++ is a reference implementation of the OME data model and
+the OME-TIFF file format for C++ developers who wish to integrate
+support for the OME data model and reading and writing the standard
+OME-TIFF file format into their software.  Potential uses include
+export of images using OME-TIFF, saving of acquired image data in
+OME-TIFF, reading metadata and image data from OME-TIFF for
+visualisation and analysis, or use of the data model metadata APIs for
+handling metadata.
+
+OME Files C++ is a re-implementation of the Bio-Formats Java API,
+providing equivalent data model, metadata and reading and writing
+interfaces.  Unlike the Bio-Formats Java API, which supports reading
+of over 130 file formats and writing of over 15 file formats at the
+time of writing, OME Files C++ restricts itself to reading and writing
+OME-TIFF and plain TIFF, and does not at present support additional
+formats.  At present it is a reference implementation for the OME-TIFF
```

Comment:
You need to get rid of one of the 'at present's here - these two sentences are a bit repetitive